File: contest/LeetCode_Weekly_Contest_202/Minimum_Operations_to_Make_Array_Equal.cpp
class Solution {
public:
    int minOperations(int n) {
        int mid = n, ret = 0;
        
        for (int i = 0; i < (n+1)>>1; ++i) {
            int x = 2 * i + 1;
            ret += mid - x;
        }
        return ret;
    }
};

File: contest/LeetCode_Weekly_Contest_202/Minimum_Operations_to_Make_Array_Equal.js
/**
 * @param {number} n
 * @return {number}
 */
var minOperations = function(n) {
    var mid = n;
    var ret = 0;
    for (var i = 0; i < (n+1)>>1; ++i) {
        ret += mid - 2*i - 1;
    }
    return ret;
};
